operantanalysis/tests/test_operantanalysis.py

```diff
@@ -0,0 +1,29 @@
+from ..operantanalysis import accessfiles, rewardretrieval, respondingduringcueanditi, leverpressing
+
+
+def test_accessfiles():
+    (subjectnumber, timecode, eventcode) = accessfiles("../operantanalysis/sampledata/!2018-11-27_08h39m.Subject _0001.txt")
+    assert len(timecode) == len(eventcode)
+    for i in timecode:
+        assert bool(i >= 0)
```

Comment:
remove `for` and use
```suggestion
    assert all(i>=0 for i in timecode)
```
and similar one below

Reply:
also you could make use of `map` if you define `nonnegative = lambda x: x>=0` and just do
```
assert all(map(nonnegative, timecode))
assert all(map(nonnegative, eventcode))
```